include/wx/osx/toolbar.h

```diff
@@ -89,9 +89,11 @@ class WXDLLIMPEXP_CORE wxToolBar: public wxToolBarBase
                                           wxObject *clientData = nullptr,
                                           const wxString& shortHelp = wxEmptyString,
                                           const wxString& longHelp = wxEmptyString) override;
+
     virtual wxToolBarToolBase *CreateTool(wxControl *control,
                                           const wxString& label) override;
 
+    void *GetMacToolbar() { return m_macToolbar; };
```

Comment:
The prevailing convention is to use `OSX` prefix, i.e.
```suggestion
    void *OSXGetToolbar() { return m_macToolbar; };
```